component/topology/syncer.go

```diff
@@ -165,3 +189,29 @@ func isContextDone(ctx context.Context) bool {
 	}
 	return false
 }
+
+// ServerInfo is server static information.
+// It will not be updated when server running.
+// So please only put static information in ServerInfo struct.
+type ServerInfo struct {
+	GitHash        string `json:"git_hash"`
+	IP             string `json:"ip"`
+	Port           uint64 `json:"listening_port"`
```

Comment:
```suggestion
	Port           uint64 `json:"port"`
```

Reply:
TiDB use `listening_port` here, I mean keep consistent with TiDB?

Reply:
Then you should call the field name as "ListeningPort". Are there specific reason that it must follow a different convention that called "Port"?